src/CFamily/Analysis/CLangAnalyzer.cs

```diff
@@ -36,7 +36,7 @@
 
 namespace SonarLint.VisualStudio.Integration.Vsix.CFamily
 {
-    internal interface ICFamilyAnalyzer : IAnalyzer
+    public interface ICFamilyAnalyzer : IAnalyzer
```

Comment:
Some types couldn't be moved from `Integration.Vsix` e.g. because of VS dependencies. So I made the types they depend on `public`.